packages/connect-node/src/transport.spec.ts

```diff
@@ -0,0 +1,107 @@
+// Copyright 2021-2024 The Connect Authors
```

Comment:
Couldn't find a good place to put this test, so created a new transport related test file.